### ISSUE TITLE
Add git to installation requirements

### ISSUE THREAD

**Issue by jdpigeon** [bcimontreal/bci_workshop#15]
git is necessary to install pygatt

**Comment by micuat** [comment]
we can probably use setup.py to avoid git
http://python-packaging.readthedocs.io/en/latest/dependencies.html#packages-not-on-pypi

**Comment by hubertjb** [comment]
Good point, people could just download a zip archive from https://github.com/peplin/pygatt and then use `pip install .` from inside the pygatt directory.

**Comment by micuat** [comment]
pointer to the section of the instruction to be revised:
https://github.com/NeuroTechX/bci-workshop/blob/master/INSTRUCTIONS.md#installation-of-additional-python-packages
